frontend/ui/data/css_tweaks.lua

```diff
@@ -1008,25 +937,27 @@ This tweak can be duplicated as a user style tweak when books contain footnotes
     margin: 0 !important;
 }
             ]],
+            separator = true,
         },
         {
-            id = "footnote-inpage_classic_classnames_smaller",
-            conflicts_with = function(id) return util.stringStartsWith(id, "footnote-inpage_") end,
-            global_conflicts_with = function(id) return util.stringStartsWith(id, "footnote-inpage_classic_classnames") end,
-            title = _("In-page classic classname footnotes (smaller)"),
+            id = "smaller_footnote-inpage",
```

Comment:
We may sort all the settings in settings.reader.lua, so it would be nice if this one started the same as the other.
I think we start with `footnote_inpage_` when targetting elements to be a footnote, and `inpage_footnote_` when adjusting a matched element, so this one could be named `inpage_footnote_smaller`.